incubator/opa/README.md

```diff
@@ -0,0 +1,57 @@
+# OPA Chart
+
+This chart will install the OPA controller and validating webhook to utilize it.  It includes a default set of policies and allows the installation of additional policy.
```

Comment:
Can you add a brief explanation of the opa and kubemgmt containers along with some high level info about the certificates for the admission controller?

Reply:
sure.